tensorflow-proto/tensorflow-proto.cabal

```diff
@@ -40,6 +40,10 @@ library
   default-language:    Haskell2010
   include-dirs: .
 
+custom-setup
+  setup-depends: Cabal
+                , proto-lens-protoc == 0.2.*
```

Comment:
Surprisingly, no, it fails in strange ways even after I enumerate the packages it complains about:
```patch
diff --git a/tensorflow-proto/tensorflow-proto.cabal b/tensorflow-proto/tensorflow-proto.cabal                                
index 9d78ba4..3c4768e 100644                                                                                                 
--- a/tensorflow-proto/tensorflow-proto.cabal                                                                                 
+++ b/tensorflow-proto/tensorflow-proto.cabal                                                                                 
@@ -35,8 +35,13 @@ library
                      , Proto.Tensorflow.Core.Framework.Versions                                                              
                      , Proto.Tensorflow.Core.Protobuf.Debug                                                                  
   build-depends:  proto-lens == 0.2.*                                                                                        
-                , proto-lens-protoc == 0.2.*                                                                                 
                 , base >= 4.7 && < 5                                                                                         
+                , bytestring                                                                                                 
+                , containers                                                                                                 
+                , data-default-class                                                                                         
+                , lens-family                                                                                                
+                , lens-labels                                                                                                
+                , text                                                                                                       
   default-language:    Haskell2010                                                                                           
   include-dirs: .                                                                                                            
```

```
$ stack --docker --docker-image=$IMAGE_NAME test
tensorflow-proto-0.1.0.0: build (lib)
tensorflow-records-0.1.0.0: test (suite: RecordsTest)
Log files have been written to: /home/gnezdo/tensorflow-haskell/.stack-work/logs/
Progress: 2/16
--  While building package tensorflow-proto-0.1.0.0 using:
      /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal\
-1.24.2.0/setup/setup --builddir=.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1.24.2.0 build lib:te\
nsorflow-proto --ghc-options " -ddump-hi -ddump-to-file"
    Process exited with code: ExitFailure 1
    Logs have been written to: /home/gnezdo/tensorflow-haskell/.stack-work/logs/tensorflow-proto-0.1.0.0.log

    Preprocessing library tensorflow-proto-0.1.0.0...
    [ 1 of 18] Compiling Proto.Tensorflow.Core.Protobuf.Debug ( .stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177\
711/Cabal-1.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs, .stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b64826\
8df668d177711/Cabal-1.24.2.0/build/Proto/Tensorflow/Core/Protobuf/Debug.o )

    /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1\
.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs:10:1: error:
	Failed to load interface for ‘Data.ProtoLens.Reexport.Data.ProtoLens’
	It is a member of the hidden package ‘proto-lens-0.2.0.1’.
	Perhaps you need to add ‘proto-lens’ to the build-depends in your .cabal file.
        Use -v to see a list of the files searched for.

    /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1\
.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs:12:1: error:
	Failed to load interface for ‘Data.ProtoLens.Reexport.Data.ProtoLens.Message.Enum’
	It is a member of the hidden package ‘proto-lens-0.2.0.1’.
	Perhaps you need to add ‘proto-lens’ to the build-depends in your .cabal file.
	Use -v to see a list of the files searched for.

    /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1\
.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs:15:1: error:
        Failed to load interface for ‘Data.ProtoLens.Reexport.Lens.Family2’
        It is a member of the hidden package ‘lens-family-1.2.1’.
	Perhaps you need to add ‘lens-family’ to the build-depends in your .cabal file.
	Use -v to see a list of the files searched for.

    /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1\
.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs:17:1: error:
	Failed to load interface for ‘Data.ProtoLens.Reexport.Lens.Family2.Unchecked’
	It is a member of the hidden package ‘lens-family-1.2.1’.
        Perhaps you need to add ‘lens-family’ to the build-depends in your .cabal file.
        Use -v to see a list of the files searched for.

    /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1\
.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs:19:1: error:
        Failed to load interface for ‘Data.ProtoLens.Reexport.Data.Default.Class’
        It is a member of the hidden package ‘data-default-class-0.1.2.0’.
        Perhaps you need to add ‘data-default-class’ to the build-depends in your .cabal file.
        Use -v to see a list of the files searched for.

    /home/gnezdo/tensorflow-haskell/tensorflow-proto/.stack-work/dist/x86_64-linux-dkd1ce2ff9c9560b648268df668d177711/Cabal-1\
.24.2.0/build/autogen/Proto/Tensorflow/Core/Protobuf/Debug.hs:21:1: error:
        Failed to load interface for ‘Data.ProtoLens.Reexport.Data.Text’
        It is a member of the hidden package ‘text-1.2.2.1’.
        Perhaps you need to add ‘text’ to the build-depends in your .cabal file.
        Use -v to see a list of the files searched for.
...
```